### PR TITLE
Blog(layout): Intrinsic layout overview

### PR DESCRIPTION
**Pull request by robdodson**
This splits #936 into a single PR for the first post and trims the text.

### CODE REVIEW

File: src/site/content/en/blog/intrinsic-layout-overview/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    whole system of layout, and how all these pieces fit together in a brand new
+    way.
+  </p>
+  <cite class="w-blockquote__cite">
+    <a href="http://www.zeldman.com/2018/05/02/transcript-intrinsic-web-design-with-jen-simmons-the-big-web-show/">Intrinsic web design with Jen Simmons</a>
+  </cite>
+</blockquote>
+
+## Why "intrinsic"? 🤔
+This may look familiar, it's the result of an _extrinsic_ approach to layout:
```
Comment (Meggin) [inline src/site/content/en/blog/intrinsic-layout-overview/index.md]: Small nit. Change comma to semi-colon.

File: src/site/content/en/blog/intrinsic-layout-overview/index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+<script async src="https://static.codepen.io/assets/embed/ei.js"></script>
+
+The article [CSS is Awesome](https://css-tricks.com/css-is-awesome/) recaps the
+frustration of this classic CSS joke really well:
+- The content doesn't shrink to fit the container
+- The container doesn't expand to fit the content
+- The container doesn't handle overflow gracefully
+
+In my experience, initially laying out a site extrinsicly is the easiest. Later,
+it becomes tedious to maintain or even becomes a problem for other folks on the
```
Comment (Meggin) [inline src/site/content/en/blog/intrinsic-layout-overview/index.md]: Suggestion to change 'a problem for other folks on the team that aren't engineers' to 'a problem for team members who try and work on the site later on'.  Just a little wary of the 'not engineer part'.

File: src/site/content/en/blog/intrinsic-layout-overview/index.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+team that aren't engineers.Perhaps you or your team have felt one or more of
```
[the remark this answers is not in the document]
Comment (Meggin) [reply src/site/content/en/blog/intrinsic-layout-overview/index.md]: +1 one to this suggestion-- I see the same missing space.

**Comment by robdodson** [comment]
@argyleink do you have a video version of that gif? I was curious if the video might be smaller in file size.

**Comment by argyleink** [comment]
i don't no.
good candidate for `loading="lazy"` though?

**Comment by robdodson** [comment]
I've converted the gifs to videos, and added a blurb for the chaos one adam mentioned. Moved the post to `_drafts` so we can land the subsequent posts and roll the series out with a good cadence.

**Pull request merged**